ws/listener.go

```diff
@@ -102,6 +100,12 @@ func (listener *wsListener) handleWebsocket(w http.ResponseWriter, r *http.Reque
 
 		connection := transporttls.NewConnection(detail, tlsConn)
 		listener.acceptF(connection) // pass the Websocket to the goroutine that will validate the HELLO handshake
+
+		// keep the Websocket alive via ping/pong control-frame msgs
+		// so it doesn't close unnecessarily thus causing ZBR to encounter
+		// unnecessary 'channel unavailable' conditions thus causing too
+		// frequent Page reboots on the client-side
```

Comment:
should this be `go connWrapper.pinger()`? I can't tell if this is already in async code

Reply:
oops, yes.  I fixed it.